package/Runtime/Scripts/Internal/NimbusAndroidAdManager.cs

```diff
@@ -34,6 +36,8 @@ internal class AdManagerListener : AndroidJavaProxy {
 			_logger.Log($"Listener Ad error: {errMessage}");
 			_adUnit.AdListenerError = new AdError(errMessage);
 			_adUnit.EmitOnAdError(_adUnit);
+			// This is where a no bid would occur and is probably checked with the following
+			// adError.Get<AndroidJavaObject>("errorType").Call<string>("name") == "NO_BID"
```

Comment:
Good to know, I think this ill be improved in V2 when requesting is handled Natively in C#